pkgs/checks/lib/src/describe.dart

```diff
@@ -13,3 +13,15 @@ String literal(Object? o) {
 }
 
 Iterable<String> indent(Iterable<String> lines) => lines.map((l) => '  $l');
+
+Iterable<String> prefixFirst(String prefix, Iterable<String> lines) sync* {
```

Comment:
at some point, doc comments on public members would be nice 🤷

Reply:
Yeah, I think this entire library will be due for polishing when we get closer to publish. I'm not sure how much churn it will see though, so I'm planning on getting the conditions fleshed out before I do that.